src/circuit.rs

```diff
@@ -987,6 +987,17 @@ mod tests {
             .unzip();
 
         let vk = VerifyingKey::build();
+
+        // Test that the pinned verification key (representing the circuit)
+        // is as expected.
+        {
+            // panic!("{:#?}", vk.vk.pinned());
+            assert_eq!(
+                format!("{:#?}", vk.vk.pinned()).as_bytes(),
+                include_bytes!("circuit_description")
```

Comment:
```suggestion
                format!("{:#?}", vk.vk.pinned()),
                include_str!("circuit_description")
```

Reply:
Good idea, though there will still be useful diffs in PRs for when `circuit_description` itself is updated, and assert_eq!() will print out the wrong string if the test fails, hence the use of the panic just before this.

I say we don't block on this.

Reply:
Yeah, I had a look around, and I couldn't find any nice crates for pretty-printing the differences between multi-line strings (`pretty_assertions` is a good starting point, but they haven't merged any of the proposed approaches for this).

I agree that simply resolving the failing test will generate git commits that provide the equivalent information.